app/api/drf/views.py

```diff
@@ -0,0 +1,106 @@
+from django.contrib.auth.models import User
+from rest_framework import viewsets
+from rest_framework.decorators import api_view
+from rest_framework.permissions import IsAuthenticated, AllowAny
+from app.api.drf.serializers import CreateUserSerializer, UserSerializer, BeerSerializer, BrewPiDeviceSerializer, FermentationProfileSerializer, FermentationProfilePointSerializer, BeerLogPointSerializer
+from app.models import Beer, BrewPiDevice, FermentationProfile, FermentationProfilePoint, BeerLogPoint
+from rest_framework import generics
+
+class CreateUserViewSet(viewsets.ModelViewSet):
```

Comment:
What's the thought with allowing users to be created without any authentication?

Although I expect the majority of Fermentrack builds to be behind a firewall, this seems pretty dangerous given that an exposed install of Fermentrack would be trivially hackable.